packages/core/contracts/test/NamespacedToModify.sol

```diff
@@ -14,6 +14,13 @@ contract Example {
         uint256 b;
     }
 
+    /// @custom:storage-location erc7201:example.with.following.comment
+    // some comment
```

Comment:
Is the intention to be a regular comment with (double) `//`? or it was supposed to be part of the NatSpec definition with (triple) `///`?

Reply:
This is intended to be a regular comment `//`, to ensure the NatSpec above the regular comment does not get deleted (as Solidity normally treats that NatSpec as part of struct).